CONTRIBUTING.md

```diff
@@ -96,9 +96,14 @@ because <additional rationale>.
 
 
 ### When adding a new Localization.lang entry
-Run `python scripts/syncLang.py -s` to search for new untranslated strings.
-If everything is OK, run `python scripts/syncLang.py -s -u` to update `JabRef_en.properties`.
-
+Add new Localization.lang("KEY") to Java file.
+Tests fail. In the test output a snippet is generated which must be added to the English translation file. There is also a snippet generated for the non-English files, but this is irrelevant.
```

Comment:
@JabRef/developers Can these irrelevant statements for the other languages be removed?